deployment_jobs/Create_Model_and_Deployment_Job.py

```diff
@@ -71,4 +62,14 @@
 )
 
 created_job = w.jobs.create(**job_settings.__dict__)
-job_id = created_job.job_id
\ No newline at end of file
+job_id = created_job.job_id
+
+# COMMAND ----------
+
+# Create registered model w/ linked deployment job
+from mlflow.tracking.client import MlflowClient
+client = MlflowClient(registry_uri="databricks-uc")
+try:
+  client.create_registered_model(model_name, deployment_job_id=job_id)
+except Exception as e:
+  print(e)
```

Comment:
```suggestion
except Exception:
  print(traceback.format_exc())
```

to print out the full traceback.